sections/5_reproducibility/reproducibility_composite_confidence_index.qmd

```diff
@@ -0,0 +1,249 @@
+---
+author:
+    - name: P. Stavropoulos
+      orcid: 0000-0003-1664-6554
+      affiliations:
+      - ref: arc
+
+affiliations:
+- id: arc
+  name: Athena Research Center
+  city: Athena
+  country: Greece
+
+title: Reuse of data in research
+---
+
+
+::: {.callout collapse="true"}
+
+
+# History
+
+| Version | Revision date | Revision    | Author              |
+|---------|---------------|-------------|---------------------|
+| 1.0     | 2023-08-25    | First draft | Petros Stavropoulos |
+
+:::
+
+# Description
+
+The **Reproducibility Composite Confidence Index (RCCI)** is a comprehensive indicator that assesses the quality, reusability, and trustworthiness of **research artefacts** (datasets, data collections, code, or software).  
+
+A high RCCI score indicates that an artefact is:  
+
+- **Highly cited** in its field (scholarly impact).  
+- **Frequently reused** by others in the scientific community.  
+- **Accepted and trusted** by peers, as reflected in citation sentiment.  
+- **Well-documented and FAIR-aligned**, with metadata enabling discoverability and reuse.  
+
+This indicator was first introduced and tested in the **TIER2 project** ([tier2-project.eu](https://tier2-project.eu/)), where it was implemented in a pilot **Reproducibility Dashboard** for funders and research-performing organisations (RPOs). The RCCI was presented and reviewed in **two stakeholder webinars** and in discussions with funders and RPOs, where feedback confirmed its value for monitoring research reproducibility.
+
+# Metrics
+
+## RCCI
+
+The RCCI integrates four dimensions into a **single score**:
+
+1. **Field-Weighted Citation Impact (FWCI)** → measures academic impact (see [Citation Impact](../2_academic_impact/citation_impact.qmd)).  
+2. **Field-Weighted Reusability Index (FWRI)** → measures how often artefacts are reused relative to others in the same field (based on [Reuse of Code in Research](../5_reproducibility/reuse_of_code_in_research.qmd) and [Reuse of Data in Research](../5_reproducibility/reuse_of_data_in_research.qmd)).  
+3. **FAIR Index (FI)** → measures metadata completeness and alignment with [FAIR data practices](../1_open_science/prevalence_open_fair_data_practices.qmd).  
+4. **Reproducibility Confidence Index (RCI)** → measures community sentiment using polarity of publications (see [Polarity of Publications](../5_reproducibility/polarity_of_publications.qmd)).  
+
+The RCCI is calculated as:
+
+$$
+RCCI = FWCI \times FWRI \times FI \times RCI
+$$
+
+A value greater than 1 (after scaling) suggests that artefacts are impactful, widely reused, FAIR-compliant, and positively regarded in the scientific community.
+
+---
+
+### Measurement
+
+#### 1. Field-Weighted Citation Impact (FWCI)
```

Comment:
It might be better if we stick to MNCS, instead of FWCI, since that aligns better with the rest of the handbook, most notably the citation impact indicator.

Reply:
I now see that in the impact of code/data, we use NCI. Also there, I think it's useful to actually use the same terminology. There are now three different names for the same thing, we should clean that up a bit.